target/linux/ath79/dts/qca9563_dlink_dir-842-c2.dts

```diff
@@ -0,0 +1,198 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca956x.dtsi"
+
+/ {
+	model = "D-Link DIR-842 C2";
+	compatible = "dlink,dir-842-c2", "qca,qca9563";
+
+	aliases {
+		led-boot = &power;
+		led-failsafe = &power;
+		led-running = &power;
+		led-upgrade = &power;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,115200n8";
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		wps {
+			label = "dir-842-c2:green:wps";
+			gpios = <&gpio 8 GPIO_ACTIVE_LOW>;
+		};
+
+		power: power {
+			label = "dir-842-c2:green:power";
+			gpios = <&gpio 15 GPIO_ACTIVE_LOW>;
+		};
+
+		internet {
+			label = "dir-842-c2:green:internet";
+			gpios = <&gpio 16 GPIO_ACTIVE_LOW>;
+		};
+
+		wlan {
+			label = "dir-842-c2:green:wlan";
+			gpios = <&gpio 19 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "phy0tpt";
+		};
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		wps {
+			linux,code = <KEY_WPS_BUTTON>;
+			gpios = <&gpio 1 GPIO_ACTIVE_LOW>;
+			debounce-interval = <60>;
+		};
+
+		reset {
+			linux,code = <KEY_RESTART>;
+			gpios = <&gpio 2 GPIO_ACTIVE_LOW>;
+			debounce-interval = <60>;
+		};
+	};
+	
+	reset-button {
+		gpio-hog;
+		output-high;
+		gpios = <11 GPIO_ACTIVE_LOW>;
+		line-name = "reset-button";
+	};
+
+};
+
+&uart {
+	status = "okay";
+};
+
+&pcie {
+	status = "okay";
+};
+
+&spi {
+	num-cs = <1>;
+
+	status = "okay";
+
+	flash@0 {
+		compatible = "jedec,spi-nor";
+		reg = <0>;
+		spi-max-frequency = <30000000>;
+
+		partitions {
+			compatible = "fixed-partitions";
+			#address-cells = <1>;
+			#size-cells = <1>;
+
+			partition@0 {
+				label = "u-boot";
+				reg = <0x000000 0x40000>;
+				read-only;
+			};
+
+			partition@40000 {
+				label = "u-boot-env";
+				reg = <0x040000 0x10000>;
+				read-only;
+			};
+
+			partition@50000 {
+				label = "devdata";
+				reg = <0x050000 0x10000>;
+				read-only;
+			};
+
+			partition@60000 {
+				label = "devconf";
+				reg = <0x060000 0x10000>;
+				read-only;
+			};
+			
+			partition@70000 {
+				label = "misc";
+				reg = <0x070000 0x10000>;
+				read-only;
+			};
+
+			partition@80000 {
+				compatible = "seama";
+				label = "firmware";
+				reg = <0x080000 0xf50000>;
+			};
+
+			art: partition@fd0000 {
+				label = "art";
+				reg = <0xfd0000 0x010000>;
+				read-only;
+			};
+			
+			partition@fe0000 {
+				label = "reserved";
+				reg = <0xfe0000 0x20000>;
+				read-only;
+			};
+			
+		};
+	};
+};
+
+&mdio0 {
+	status = "okay";
+
+	phy-mask = <0>;
+
+	phy0: ethernet-phy@0 {
+		reg = <0>;
+		phy-mode = "sgmii";
```

Comment:
Looking at the ar8327 driver, we should not need this property, PHY mode for the individual ports are set thru `qca,ar8327-initvals` property.

Can yo ureport back if it still works without the `phy-mode` here?

Reply:
The dts wont compile after I removed that...

Reply:
That sounds strange. It should compile even in case the driver needs this property. What was the exact error? Build with `make -j1 V=s` to see the error.

Reply:
```
ake[5]: Entering directory '/home/ha/gitopenwrt2/openwrt/target/linux/ath79/image'
make[5]: Nothing to be done for 'compile'.
mips-openwrt-linux-musl-cpp -nostdinc -x assembler-with-cpp -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/arch/mips/boot/dts -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/arch/mips/boot/dts/include -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/include/ -undef -D__DTS__  -o /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-ar9331_8dev_carambola2.dtb.tmp ../dts/ar9331_8dev_carambola2.dts
/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/scripts/dtc/dtc -O dtb -i../dts/ -Wno-unit_address_vs_reg -Wno-unit_address_vs_reg -Wno-simple_bus_reg -Wno-unit_address_format -Wno-pci_bridge -Wno-pci_device_bus_num -Wno-pci_device_reg -Wno-avoid_unnecessary_addr_size -Wno-alias_paths -Wno-graph_child_address -Wno-graph_port -Wno-unique_unit_address  -o /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-ar9331_8dev_carambola2.dtb /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-ar9331_8dev_carambola2.dtb.tmp
rm -f /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-ar9331_8dev_carambola2.dtb.tmp
mips-openwrt-linux-musl-cpp -nostdinc -x assembler-with-cpp -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/arch/mips/boot/dts -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/arch/mips/boot/dts/include -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/include/ -undef -D__DTS__  -o /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-ar7161_adtran_bsap1800-v2.dtb.tmp ../dts/ar7161_adtran_bsap1800-v2.dts
/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/scripts/dtc/dtc -O dtb -i../dts/ -Wno-unit_address_vs_reg -Wno-unit_address_vs_reg -Wno-simple_bus_reg -Wno-unit_address_format -Wno-pci_bridge -Wno-pci_device_bus_num -Wno-pci_device_reg -Wno-avoid_unnecessary_addr_size -Wno-alias_paths -Wno-graph_child_address -Wno-graph_port -Wno-unique_unit_address  -o /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-ar7161_adtran_bsap1800-v2.dtb /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-ar7161_adtran_bsap1800-v2.dtb.tmp
rm -f /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-ar7161_adtran_bsap1800-v2.dtb.tmp
mips-openwrt-linux-musl-cpp -nostdinc -x assembler-with-cpp -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/arch/mips/boot/dts -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/arch/mips/boot/dts/include -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/include/ -undef -D__DTS__  -o /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-ar7161_adtran_bsap1840.dtb.tmp ../dts/ar7161_adtran_bsap1840.dts
/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/scripts/dtc/dtc -O dtb -i../dts/ -Wno-unit_address_vs_reg -Wno-unit_address_vs_reg -Wno-simple_bus_reg -Wno-unit_address_format -Wno-pci_bridge -Wno-pci_device_bus_num -Wno-pci_device_reg -Wno-avoid_unnecessary_addr_size -Wno-alias_paths -Wno-graph_child_address -Wno-graph_port -Wno-unique_unit_address  -o /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-ar7161_adtran_bsap1840.dtb /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-ar7161_adtran_bsap1840.dtb.tmp
rm -f /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-ar7161_adtran_bsap1840.dtb.tmp
mips-openwrt-linux-musl-cpp -nostdinc -x assembler-with-cpp -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/arch/mips/boot/dts -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/arch/mips/boot/dts/include -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/include/ -undef -D__DTS__  -o /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-ar7161_aruba_ap-105.dtb.tmp ../dts/ar7161_aruba_ap-105.dts
/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/scripts/dtc/dtc -O dtb -i../dts/ -Wno-unit_address_vs_reg -Wno-unit_address_vs_reg -Wno-simple_bus_reg -Wno-unit_address_format -Wno-pci_bridge -Wno-pci_device_bus_num -Wno-pci_device_reg -Wno-avoid_unnecessary_addr_size -Wno-alias_paths -Wno-graph_child_address -Wno-graph_port -Wno-unique_unit_address  -o /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-ar7161_aruba_ap-105.dtb /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-ar7161_aruba_ap-105.dtb.tmp
rm -f /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-ar7161_aruba_ap-105.dtb.tmp
mips-openwrt-linux-musl-cpp -nostdinc -x assembler-with-cpp -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/arch/mips/boot/dts -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/arch/mips/boot/dts/include -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/include/ -undef -D__DTS__  -o /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-ar7242_avm_fritz300e.dtb.tmp ../dts/ar7242_avm_fritz300e.dts
/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/scripts/dtc/dtc -O dtb -i../dts/ -Wno-unit_address_vs_reg -Wno-unit_address_vs_reg -Wno-simple_bus_reg -Wno-unit_address_format -Wno-pci_bridge -Wno-pci_device_bus_num -Wno-pci_device_reg -Wno-avoid_unnecessary_addr_size -Wno-alias_paths -Wno-graph_child_address -Wno-graph_port -Wno-unique_unit_address  -o /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-ar7242_avm_fritz300e.dtb /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-ar7242_avm_fritz300e.dtb.tmp
rm -f /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-ar7242_avm_fritz300e.dtb.tmp
mips-openwrt-linux-musl-cpp -nostdinc -x assembler-with-cpp -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/arch/mips/boot/dts -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/arch/mips/boot/dts/include -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/include/ -undef -D__DTS__  -o /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-qca9561_avm_fritz4020.dtb.tmp ../dts/qca9561_avm_fritz4020.dts
/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/scripts/dtc/dtc -O dtb -i../dts/ -Wno-unit_address_vs_reg -Wno-unit_address_vs_reg -Wno-simple_bus_reg -Wno-unit_address_format -Wno-pci_bridge -Wno-pci_device_bus_num -Wno-pci_device_reg -Wno-avoid_unnecessary_addr_size -Wno-alias_paths -Wno-graph_child_address -Wno-graph_port -Wno-unique_unit_address  -o /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-qca9561_avm_fritz4020.dtb /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-qca9561_avm_fritz4020.dtb.tmp
rm -f /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-qca9561_avm_fritz4020.dtb.tmp
mips-openwrt-linux-musl-cpp -nostdinc -x assembler-with-cpp -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/arch/mips/boot/dts -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/arch/mips/boot/dts/include -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/include/ -undef -D__DTS__  -o /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-ar7242_buffalo_bhr-4grv.dtb.tmp ../dts/ar7242_buffalo_bhr-4grv.dts
/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/scripts/dtc/dtc -O dtb -i../dts/ -Wno-unit_address_vs_reg -Wno-unit_address_vs_reg -Wno-simple_bus_reg -Wno-unit_address_format -Wno-pci_bridge -Wno-pci_device_bus_num -Wno-pci_device_reg -Wno-avoid_unnecessary_addr_size -Wno-alias_paths -Wno-graph_child_address -Wno-graph_port -Wno-unique_unit_address  -o /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-ar7242_buffalo_bhr-4grv.dtb /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-ar7242_buffalo_bhr-4grv.dtb.tmp
rm -f /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-ar7242_buffalo_bhr-4grv.dtb.tmp
mips-openwrt-linux-musl-cpp -nostdinc -x assembler-with-cpp -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/arch/mips/boot/dts -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/arch/mips/boot/dts/include -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/include/ -undef -D__DTS__  -o /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-qca9557_buffalo_bhr-4grv2.dtb.tmp ../dts/qca9557_buffalo_bhr-4grv2.dts
/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/scripts/dtc/dtc -O dtb -i../dts/ -Wno-unit_address_vs_reg -Wno-unit_address_vs_reg -Wno-simple_bus_reg -Wno-unit_address_format -Wno-pci_bridge -Wno-pci_device_bus_num -Wno-pci_device_reg -Wno-avoid_unnecessary_addr_size -Wno-alias_paths -Wno-graph_child_address -Wno-graph_port -Wno-unique_unit_address  -o /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-qca9557_buffalo_bhr-4grv2.dtb /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-qca9557_buffalo_bhr-4grv2.dtb.tmp
rm -f /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-qca9557_buffalo_bhr-4grv2.dtb.tmp
mips-openwrt-linux-musl-cpp -nostdinc -x assembler-with-cpp -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/arch/mips/boot/dts -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/arch/mips/boot/dts/include -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/include/ -undef -D__DTS__  -o /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-ar7161_buffalo_wzr-hp-ag300h.dtb.tmp ../dts/ar7161_buffalo_wzr-hp-ag300h.dts
/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/scripts/dtc/dtc -O dtb -i../dts/ -Wno-unit_address_vs_reg -Wno-unit_address_vs_reg -Wno-simple_bus_reg -Wno-unit_address_format -Wno-pci_bridge -Wno-pci_device_bus_num -Wno-pci_device_reg -Wno-avoid_unnecessary_addr_size -Wno-alias_paths -Wno-graph_child_address -Wno-graph_port -Wno-unique_unit_address  -o /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-ar7161_buffalo_wzr-hp-ag300h.dtb /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-ar7161_buffalo_wzr-hp-ag300h.dtb.tmp
rm -f /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-ar7161_buffalo_wzr-hp-ag300h.dtb.tmp
mips-openwrt-linux-musl-cpp -nostdinc -x assembler-with-cpp -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/arch/mips/boot/dts -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/arch/mips/boot/dts/include -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/include/ -undef -D__DTS__  -o /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-ar7242_buffalo_wzr-hp-g302h-a1a0.dtb.tmp ../dts/ar7242_buffalo_wzr-hp-g302h-a1a0.dts
/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/scripts/dtc/dtc -O dtb -i../dts/ -Wno-unit_address_vs_reg -Wno-unit_address_vs_reg -Wno-simple_bus_reg -Wno-unit_address_format -Wno-pci_bridge -Wno-pci_device_bus_num -Wno-pci_device_reg -Wno-avoid_unnecessary_addr_size -Wno-alias_paths -Wno-graph_child_address -Wno-graph_port -Wno-unique_unit_address  -o /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-ar7242_buffalo_wzr-hp-g302h-a1a0.dtb /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-ar7242_buffalo_wzr-hp-g302h-a1a0.dtb.tmp
rm -f /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-ar7242_buffalo_wzr-hp-g302h-a1a0.dtb.tmp
mips-openwrt-linux-musl-cpp -nostdinc -x assembler-with-cpp -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/arch/mips/boot/dts -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/arch/mips/boot/dts/include -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/include/ -undef -D__DTS__  -o /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-ar7242_buffalo_wzr-hp-g450h.dtb.tmp ../dts/ar7242_buffalo_wzr-hp-g450h.dts
/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/scripts/dtc/dtc -O dtb -i../dts/ -Wno-unit_address_vs_reg -Wno-unit_address_vs_reg -Wno-simple_bus_reg -Wno-unit_address_format -Wno-pci_bridge -Wno-pci_device_bus_num -Wno-pci_device_reg -Wno-avoid_unnecessary_addr_size -Wno-alias_paths -Wno-graph_child_address -Wno-graph_port -Wno-unique_unit_address  -o /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-ar7242_buffalo_wzr-hp-g450h.dtb /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-ar7242_buffalo_wzr-hp-g450h.dtb.tmp
rm -f /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-ar7242_buffalo_wzr-hp-g450h.dtb.tmp
mips-openwrt-linux-musl-cpp -nostdinc -x assembler-with-cpp -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/arch/mips/boot/dts -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/arch/mips/boot/dts/include -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/include/ -undef -D__DTS__  -o /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-qca9533_comfast_cf-e110n-v2.dtb.tmp ../dts/qca9533_comfast_cf-e110n-v2.dts
/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/scripts/dtc/dtc -O dtb -i../dts/ -Wno-unit_address_vs_reg -Wno-unit_address_vs_reg -Wno-simple_bus_reg -Wno-unit_address_format -Wno-pci_bridge -Wno-pci_device_bus_num -Wno-pci_device_reg -Wno-avoid_unnecessary_addr_size -Wno-alias_paths -Wno-graph_child_address -Wno-graph_port -Wno-unique_unit_address  -o /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-qca9533_comfast_cf-e110n-v2.dtb /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-qca9533_comfast_cf-e110n-v2.dtb.tmp
rm -f /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-qca9533_comfast_cf-e110n-v2.dtb.tmp
mips-openwrt-linux-musl-cpp -nostdinc -x assembler-with-cpp -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/arch/mips/boot/dts -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/arch/mips/boot/dts/include -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/include/ -undef -D__DTS__  -o /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-ar9344_comfast_cf-e120a-v3.dtb.tmp ../dts/ar9344_comfast_cf-e120a-v3.dts
/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/scripts/dtc/dtc -O dtb -i../dts/ -Wno-unit_address_vs_reg -Wno-unit_address_vs_reg -Wno-simple_bus_reg -Wno-unit_address_format -Wno-pci_bridge -Wno-pci_device_bus_num -Wno-pci_device_reg -Wno-avoid_unnecessary_addr_size -Wno-alias_paths -Wno-graph_child_address -Wno-graph_port -Wno-unique_unit_address  -o /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-ar9344_comfast_cf-e120a-v3.dtb /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-ar9344_comfast_cf-e120a-v3.dtb.tmp
rm -f /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-ar9344_comfast_cf-e120a-v3.dtb.tmp
mips-openwrt-linux-musl-cpp -nostdinc -x assembler-with-cpp -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/arch/mips/boot/dts -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/arch/mips/boot/dts/include -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/include/ -undef -D__DTS__  -o /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-qca9531_comfast_cf-e5.dtb.tmp ../dts/qca9531_comfast_cf-e5.dts
/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/scripts/dtc/dtc -O dtb -i../dts/ -Wno-unit_address_vs_reg -Wno-unit_address_vs_reg -Wno-simple_bus_reg -Wno-unit_address_format -Wno-pci_bridge -Wno-pci_device_bus_num -Wno-pci_device_reg -Wno-avoid_unnecessary_addr_size -Wno-alias_paths -Wno-graph_child_address -Wno-graph_port -Wno-unique_unit_address  -o /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-qca9531_comfast_cf-e5.dtb /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-qca9531_comfast_cf-e5.dtb.tmp
rm -f /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-qca9531_comfast_cf-e5.dtb.tmp
mips-openwrt-linux-musl-cpp -nostdinc -x assembler-with-cpp -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/arch/mips/boot/dts -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/arch/mips/boot/dts/include -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/include/ -undef -D__DTS__  -o /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-qca9558_devolo_dvl1200e.dtb.tmp ../dts/qca9558_devolo_dvl1200e.dts
/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/scripts/dtc/dtc -O dtb -i../dts/ -Wno-unit_address_vs_reg -Wno-unit_address_vs_reg -Wno-simple_bus_reg -Wno-unit_address_format -Wno-pci_bridge -Wno-pci_device_bus_num -Wno-pci_device_reg -Wno-avoid_unnecessary_addr_size -Wno-alias_paths -Wno-graph_child_address -Wno-graph_port -Wno-unique_unit_address  -o /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-qca9558_devolo_dvl1200e.dtb /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-qca9558_devolo_dvl1200e.dtb.tmp
rm -f /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-qca9558_devolo_dvl1200e.dtb.tmp
mips-openwrt-linux-musl-cpp -nostdinc -x assembler-with-cpp -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/arch/mips/boot/dts -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/arch/mips/boot/dts/include -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/include/ -undef -D__DTS__  -o /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-qca9558_devolo_dvl1200i.dtb.tmp ../dts/qca9558_devolo_dvl1200i.dts
/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/scripts/dtc/dtc -O dtb -i../dts/ -Wno-unit_address_vs_reg -Wno-unit_address_vs_reg -Wno-simple_bus_reg -Wno-unit_address_format -Wno-pci_bridge -Wno-pci_device_bus_num -Wno-pci_device_reg -Wno-avoid_unnecessary_addr_size -Wno-alias_paths -Wno-graph_child_address -Wno-graph_port -Wno-unique_unit_address  -o /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-qca9558_devolo_dvl1200i.dtb /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-qca9558_devolo_dvl1200i.dtb.tmp
rm -f /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-qca9558_devolo_dvl1200i.dtb.tmp
mips-openwrt-linux-musl-cpp -nostdinc -x assembler-with-cpp -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/arch/mips/boot/dts -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/arch/mips/boot/dts/include -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/include/ -undef -D__DTS__  -o /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-qca9558_devolo_dvl1750c.dtb.tmp ../dts/qca9558_devolo_dvl1750c.dts
/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/scripts/dtc/dtc -O dtb -i../dts/ -Wno-unit_address_vs_reg -Wno-unit_address_vs_reg -Wno-simple_bus_reg -Wno-unit_address_format -Wno-pci_bridge -Wno-pci_device_bus_num -Wno-pci_device_reg -Wno-avoid_unnecessary_addr_size -Wno-alias_paths -Wno-graph_child_address -Wno-graph_port -Wno-unique_unit_address  -o /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-qca9558_devolo_dvl1750c.dtb /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-qca9558_devolo_dvl1750c.dtb.tmp
rm -f /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-qca9558_devolo_dvl1750c.dtb.tmp
mips-openwrt-linux-musl-cpp -nostdinc -x assembler-with-cpp -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/arch/mips/boot/dts -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/arch/mips/boot/dts/include -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/include/ -undef -D__DTS__  -o /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-qca9558_devolo_dvl1750e.dtb.tmp ../dts/qca9558_devolo_dvl1750e.dts
/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/scripts/dtc/dtc -O dtb -i../dts/ -Wno-unit_address_vs_reg -Wno-unit_address_vs_reg -Wno-simple_bus_reg -Wno-unit_address_format -Wno-pci_bridge -Wno-pci_device_bus_num -Wno-pci_device_reg -Wno-avoid_unnecessary_addr_size -Wno-alias_paths -Wno-graph_child_address -Wno-graph_port -Wno-unique_unit_address  -o /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-qca9558_devolo_dvl1750e.dtb /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-qca9558_devolo_dvl1750e.dtb.tmp
rm -f /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-qca9558_devolo_dvl1750e.dtb.tmp
mips-openwrt-linux-musl-cpp -nostdinc -x assembler-with-cpp -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/arch/mips/boot/dts -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/arch/mips/boot/dts/include -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/include/ -undef -D__DTS__  -o /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-qca9558_devolo_dvl1750i.dtb.tmp ../dts/qca9558_devolo_dvl1750i.dts
/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/scripts/dtc/dtc -O dtb -i../dts/ -Wno-unit_address_vs_reg -Wno-unit_address_vs_reg -Wno-simple_bus_reg -Wno-unit_address_format -Wno-pci_bridge -Wno-pci_device_bus_num -Wno-pci_device_reg -Wno-avoid_unnecessary_addr_size -Wno-alias_paths -Wno-graph_child_address -Wno-graph_port -Wno-unique_unit_address  -o /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-qca9558_devolo_dvl1750i.dtb /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-qca9558_devolo_dvl1750i.dtb.tmp
rm -f /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-qca9558_devolo_dvl1750i.dtb.tmp
mips-openwrt-linux-musl-cpp -nostdinc -x assembler-with-cpp -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/arch/mips/boot/dts -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/arch/mips/boot/dts/include -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/include/ -undef -D__DTS__  -o /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-qca9558_devolo_dvl1750x.dtb.tmp ../dts/qca9558_devolo_dvl1750x.dts
/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/scripts/dtc/dtc -O dtb -i../dts/ -Wno-unit_address_vs_reg -Wno-unit_address_vs_reg -Wno-simple_bus_reg -Wno-unit_address_format -Wno-pci_bridge -Wno-pci_device_bus_num -Wno-pci_device_reg -Wno-avoid_unnecessary_addr_size -Wno-alias_paths -Wno-graph_child_address -Wno-graph_port -Wno-unique_unit_address  -o /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-qca9558_devolo_dvl1750x.dtb /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-qca9558_devolo_dvl1750x.dtb.tmp
rm -f /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-qca9558_devolo_dvl1750x.dtb.tmp
mips-openwrt-linux-musl-cpp -nostdinc -x assembler-with-cpp -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/arch/mips/boot/dts -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/arch/mips/boot/dts/include -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/include/ -undef -D__DTS__  -o /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-ar7161_dlink_dir-825-b1.dtb.tmp ../dts/ar7161_dlink_dir-825-b1.dts
/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/scripts/dtc/dtc -O dtb -i../dts/ -Wno-unit_address_vs_reg -Wno-unit_address_vs_reg -Wno-simple_bus_reg -Wno-unit_address_format -Wno-pci_bridge -Wno-pci_device_bus_num -Wno-pci_device_reg -Wno-avoid_unnecessary_addr_size -Wno-alias_paths -Wno-graph_child_address -Wno-graph_port -Wno-unique_unit_address  -o /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-ar7161_dlink_dir-825-b1.dtb /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-ar7161_dlink_dir-825-b1.dtb.tmp
rm -f /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-ar7161_dlink_dir-825-b1.dtb.tmp
mips-openwrt-linux-musl-cpp -nostdinc -x assembler-with-cpp -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/arch/mips/boot/dts -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/arch/mips/boot/dts/include -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/include/ -undef -D__DTS__  -o /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-ar9344_dlink_dir-825-c1.dtb.tmp ../dts/ar9344_dlink_dir-825-c1.dts
/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/scripts/dtc/dtc -O dtb -i../dts/ -Wno-unit_address_vs_reg -Wno-unit_address_vs_reg -Wno-simple_bus_reg -Wno-unit_address_format -Wno-pci_bridge -Wno-pci_device_bus_num -Wno-pci_device_reg -Wno-avoid_unnecessary_addr_size -Wno-alias_paths -Wno-graph_child_address -Wno-graph_port -Wno-unique_unit_address  -o /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-ar9344_dlink_dir-825-c1.dtb /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-ar9344_dlink_dir-825-c1.dtb.tmp
rm -f /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-ar9344_dlink_dir-825-c1.dtb.tmp
mips-openwrt-linux-musl-cpp -nostdinc -x assembler-with-cpp -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/arch/mips/boot/dts -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/arch/mips/boot/dts/include -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/include/ -undef -D__DTS__  -o /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-ar9344_dlink_dir-835-a1.dtb.tmp ../dts/ar9344_dlink_dir-835-a1.dts
/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/scripts/dtc/dtc -O dtb -i../dts/ -Wno-unit_address_vs_reg -Wno-unit_address_vs_reg -Wno-simple_bus_reg -Wno-unit_address_format -Wno-pci_bridge -Wno-pci_device_bus_num -Wno-pci_device_reg -Wno-avoid_unnecessary_addr_size -Wno-alias_paths -Wno-graph_child_address -Wno-graph_port -Wno-unique_unit_address  -o /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-ar9344_dlink_dir-835-a1.dtb /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-ar9344_dlink_dir-835-a1.dtb.tmp
rm -f /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-ar9344_dlink_dir-835-a1.dtb.tmp
mips-openwrt-linux-musl-cpp -nostdinc -x assembler-with-cpp -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/arch/mips/boot/dts -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/arch/mips/boot/dts/include -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/include/ -undef -D__DTS__  -o /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-qca9563_dlink_dir-859-a1.dtb.tmp ../dts/qca9563_dlink_dir-859-a1.dts
/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/scripts/dtc/dtc -O dtb -i../dts/ -Wno-unit_address_vs_reg -Wno-unit_address_vs_reg -Wno-simple_bus_reg -Wno-unit_address_format -Wno-pci_bridge -Wno-pci_device_bus_num -Wno-pci_device_reg -Wno-avoid_unnecessary_addr_size -Wno-alias_paths -Wno-graph_child_address -Wno-graph_port -Wno-unique_unit_address  -o /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-qca9563_dlink_dir-859-a1.dtb /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-qca9563_dlink_dir-859-a1.dtb.tmp
rm -f /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-qca9563_dlink_dir-859-a1.dtb.tmp
mips-openwrt-linux-musl-cpp -nostdinc -x assembler-with-cpp -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/arch/mips/boot/dts -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/arch/mips/boot/dts/include -I/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/include/ -undef -D__DTS__  -o /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-qca9563_dlink_dir-842-c2.dtb.tmp ../dts/qca9563_dlink_dir-842-c2.dts
/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/linux-4.19.57/scripts/dtc/dtc -O dtb -i../dts/ -Wno-unit_address_vs_reg -Wno-unit_address_vs_reg -Wno-simple_bus_reg -Wno-unit_address_format -Wno-pci_bridge -Wno-pci_device_bus_num -Wno-pci_device_reg -Wno-avoid_unnecessary_addr_size -Wno-alias_paths -Wno-graph_child_address -Wno-graph_port -Wno-unique_unit_address  -o /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-qca9563_dlink_dir-842-c2.dtb /home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-qca9563_dlink_dir-842-c2.dtb.tmp
/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-qca9563_dlink_dir-842-c2.dtb: ERROR (phandle_references): /ahb/eth@19000000: Reference to non-existent node or label "phy0"

ERROR: Input tree has errors, aborting (use -f to force output)
Makefile:82: recipe for target '/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-qca9563_dlink_dir-842-c2.dtb' failed
make[5]: *** [/home/ha/gitopenwrt2/openwrt/build_dir/target-mips_24kc_musl/linux-ath79_generic/image-qca9563_dlink_dir-842-c2.dtb] Error 2
make[5]: Leaving directory '/home/ha/gitopenwrt2/openwrt/target/linux/ath79/image'
Makefile:19: recipe for target 'install' failed
make[4]: *** [install] Error 2
make[4]: Leaving directory '/home/ha/gitopenwrt2/openwrt/target/linux/ath79'
Makefile:13: recipe for target 'install' failed
make[3]: *** [install] Error 2
make[3]: Leaving directory '/home/ha/gitopenwrt2/openwrt/target/linux'
time: target/linux/install#11.03#2.32#13.11
target/Makefile:23: recipe for target 'target/linux/install' failed
make[2]: *** [target/linux/install] Error 2
make[2]: Leaving directory '/home/ha/gitopenwrt2/openwrt'
target/Makefile:19: recipe for target '/home/ha/gitopenwrt2/openwrt/staging_dir/target-mips_24kc_musl/stamp/.target_install' failed
make[1]: *** [/home/ha/gitopenwrt2/openwrt/staging_dir/target-mips_24kc_musl/stamp/.target_install] Error 2
make[1]: Leaving directory '/home/ha/gitopenwrt2/openwrt'
/home/ha/gitopenwrt2/openwrt/include/toplevel.mk:216: recipe for target 'world' failed
make: *** [world] Error 2

```
This is the error message

Reply:
`ERROR (phandle_references): /ahb/eth@19000000: Reference to non-existent node or label "phy0"` suggests you removed more than just line 156 (`phy-mode = "sgmii";`).

The `phy0` handle is supposed to be untouched as it's referenced from the `phy-mode` property of the ethernet node (which shall be left untouched).

Reply:
sorry of my mistake, removed it, compiled, and everything still works fine, I already committed the change